docs/09.other_bots.md

```diff
@@ -15,12 +15,18 @@ or Gunicorn, or even as a FastAPI sub-application.
 
 More information on this can be found in [Yuyo's guide](https://yuyo.cursed.solutions/usage/asgi/).
```

Comment:
Make this a community resources page and put all this stuff under a "other bot implementations" sub section

+ Mention stuff like yuyo and sake